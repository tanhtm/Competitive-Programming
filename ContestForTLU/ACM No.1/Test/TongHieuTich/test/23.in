90
837 462
104 789
-36 823
-695 659
-185 -285
891 434
210 -524
543 273
86 433
573 292
114 -498
-509 146
-782 802
235 -735
258 -76
-348 227
11 -6
-297 -427
276 -650
-147 923
724 -94
828 295
-370 895
473 821
-826 177
-955 665
-314 805
727 -250
343 -45
-691 567
813 545
365 -334
-862 889
905 900
56 -820
69 123
-519 749
315 241
137 750
435 -233
-473 -410
-309 -566
386 -330
452 -487
-235 852
445 799
-900 -141
374 -532
941 -445
362 874
203 -518
356 473
676 -505
655 -870
10 190
721 219
-663 859
733 283
152 27
499 619
-123 940
-809 -50
591 272
-491 189
399 641
-923 -356
382 710
328 29
968 -404
-806 -317
940 -554
94 -438
430 167
-522 -744
-805 -710
-739 -246
-649 -921
-589 -835
-740 964
-401 -596
-114 -650
745 312
373 82
327 670
942 -58
-244 334
297 71
-18 842
-844 -764
427 712
