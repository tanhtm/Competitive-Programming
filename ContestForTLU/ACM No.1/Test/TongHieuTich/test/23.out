121781
-152926
-169008
12186
-11750
151381
-57619
55080
-45023
60766
-58752
59441
-7920
-121250
15197
17393
21
-23530
-86581
-207580
128835
149639
-166031
-112578
162736
117450
-137357
116507
28906
38998
90986
5417
-11819
2256
-167316
-2592
-72910
10286
-135932
33734
13907
-56218
10024
-8216
-167669
-110094
197529
-35787
171864
-158208
-56778
-24248
50487
-81968
-9000
117970
-74578
114300
5593
-33540
-217117
162995
68824
51340
-62920
181298
-89544
26685
193452
137286
144171
-45752
39252
-70263
35981
121401
-106760
-87576
-95424
-48603
-102376
114420
33101
-85492
221000
-13005
20792
-177160
32160
-81153
